enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.ts

```diff
@@ -25,28 +21,13 @@ import registerVisualizations from "metabase/visualizations/register";
 const registerVisualizationsOnce = _.once(registerVisualizations);
 
 interface InitDataLoaderParameters {
-  config: SDKConfigType;
+  config: SDKConfig;
 }
 
-const getErrorMessage = (authType: SDKConfigType["authType"]) => {
-  if (authType === "jwt") {
-    return t`Could not authenticate: invalid JWT URI or JWT provider did not return a valid JWT token`;
-  }
-
-  if (authType === "apiKey") {
-    return t`Could not authenticate: invalid API key`;
-  }
-
-  return t`Invalid auth type`;
-};
-
-const isJwtAuth = (config: SDKConfigType): config is SdkConfigWithJWT =>
-  config.authType === "jwt" && !!config.jwtProviderUri;
+const isValidJwtAuth = (config: SDKConfig): config is SDKConfig =>
```

Comment:
I think we can also safely remove `config is SDKConfig` because the type signature is `config: SDKConfig`